AjaxControlToolkit.Jasmine/Global.asax.cs

```diff
@@ -10,6 +11,7 @@ public class Global : System.Web.HttpApplication {
             RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
 
+            AjaxFileUploadHelper.RootTempFolderPath = HttpContext.Current.Server.MapPath("~/Temp");
```

Comment:
Move `MapPath` call to `BuildRootTempFolder`, so that end user can specify plain "~/Temp". Same for SampleSite.